lib/whitehall_importer/integrity_checker.rb

```diff
@@ -58,6 +58,25 @@ def content_problems
       problems
     end
 
+    def time_problems
+      problems = []
+
+      proposed_first_published_at = proposed_payload["first_published_at"]
+      first_published_at = publishing_api_content["details"]["first_public_at"]
```

Comment:
Can we add something in the commit message about why we are using first_public_at? It'd be good to have that decision documented.

Reply:
Sure